actions/gerald/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Gerald
```

Comment:
Gerald has a few different actions - I think we may want to call this "gerald-pr" - there would also be a "gerald-comments" and "gerald-push".